index.bs

```diff
@@ -113,186 +113,206 @@ div.enum-description > table > thead > tr > th {
       finer-grained autoplay control.
     </div>
 
-    <div class="note">
-      Note: In addition, document's {{Document/autoplayPolicy}} can also be
-      used to know whether [[webaudio inline]] can be allowed to start if a
-      user-agent blocks web audio by default.
-    </div>
-
-    <div class=example>
-      [Web Audio](https://webaudio.github.io/web-audio-api/#allowed-to-start)
-      uses [=sticky activation=] to determine if {{AudioContext}} can be allowed
-      to start. If the document is allowed to autoplay, then {{AudioContext}}
-      should also be allowed to start.
-
-      <pre class="lang-javascript">
-        var ac = new AudioContext;
-        if (document.autoplayPolicy === "allowed") {
-          ac.onstatechange = function() {
-            if (ac.state === "running") {
-              // Start running audio app
-            } else {
-              // Display a bit of UI to ask the user to start the audio app.
-              // Audio starts via calling ac.resume() from a handler, and
-              // 'onstatechange' allows knowing when the audio stack is ready.
-            }
-          }
-        }
-      </pre>
-    </div>
-
-  <h3 id="media-element-api">The HTMLMediaElement API</h3>
+  <h4 id="query-by-a-element">Query by a Element</h3>
     <pre class="idl">
-    partial interface HTMLMediaElement {
-      readonly attribute AutoplayPolicy autoplayPolicy;
-    };
+      [Exposed=Window]
+      partial interface Navigator {
+        AutoplayPolicy getAutoplayPolicy(HTMLMediaElement element);
+
+        AutoplayPolicy getAutoplayPolicy(AudioContext context);
+      };
     </pre>
-    This represents the current status of whether this {{HTMLMediaElement}} is
+    This represents the current status of whether the queried element is
     allowed to autoplay or not.
 
     <dl class="switch">
       <dt>If the value is {{allowed}}</dt>
       <dd>
-        This media element is allowed to autoplay within the current execution
+        This element is allowed to autoplay within the current execution
         context.
       </dd>
 
       <dt>If the value is {{allowed-muted}}</dt>
       <dd>
-        This media element will only be allowed to autoplay if it's an
-        {{inaudible media element}}. If authors make this media element audible
-        right after the media element started playing, then the user-agent
-        **MUST** pause this media element immediately because it's no longer
-        inaudible.
+        This element will only be allowed to autoplay if it's inaudible.
+
+        <div class="note">
+          Note: Currently, this attribute will only be returned when a queried
+          constructor is the type of {{HTMLMediaElement}}. The inaudible media
+          means {{inaudible media element}}.
+
+          In addition, if authors make an inaudible media element audible
+          right after it starts playing, then the user-agent **MUST** pause
```

Comment:
perhaps "may" instead of "must"?  The UA should retain considerable freedom about the particular autoplay policy it chooses to implement.

Reply:
Okay, will change this part to a recommandation for a user agent. Thanks.